importccl: fix INSERT with missing columns with IMPORT PGDUMP

otan opened the pull request:
We've introduced the functionality in v20.2 to INSERT PGDUMP with
less columns than there are total. As a result, we've introduced a bug
where missing a column in the INSERT would result in a panic.

Release note (bug fix): Fixed a bug where a IMPORTING a PGDUMP with
INSERTs not targeting all columns in the database would panic.

cockroach-teamcity commented:
<!-- Reviewable:start -->
This change is [<img src="https://reviewable.io/review_button.png" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/cockroachdb/cockroach/55921)
<!-- Reviewable:end -->

otan committed:
importccl: fix INSERT with missing columns with IMPORT PGDUMP

We've introduced the functionality in v20.2 to INSERT PGDUMP with
less columns than there are total. As a result, we've introduced a bug
where missing a column in the INSERT would result in a panic.

Release note (bug fix): Fixed a bug where a IMPORTING a PGDUMP with
INSERTs not targeting all columns in the database would panic.

otan commented:
thanks!

bors r=dt
